sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -58,7 +58,6 @@
 #define SOF_SSP_BT_OFFLOAD_PRESENT		BIT(22)
 #define SOF_RT5682S_HEADPHONE_CODEC_PRESENT	BIT(23)
 #define SOF_MAX98390_SPEAKER_AMP_PRESENT	BIT(24)
-#define SOF_MAX98390_TWEETER_SPEAKER_PRESENT	BIT(25)
```

Comment:
why is this changed?

You are doing too many changes in this patch, it's impossible to review.

If you move stuff around, just move stuff and don't mix code moves with functionality changes.